### PR TITLE
Admin cards: Rename "FIRST Grants" to "Hackathon Grants"

### PR DESCRIPTION
**Pull request by aramshiva**
Solves #4783. Renames all Hackathon Grants to remove FIRST's affilation on admin menu.

### CODE REVIEW

File: app/javascript/components/command_bar/CommandBar.js [hunk 1]
```diff
@@ -343,7 +343,7 @@ export default function CommandBar({admin = false, adminUrls = {}}) {
       id: "admin_tool_13",
       section: "Admin Tools",
       priority: Priority.HIGH,
-      name: "FIRST Grants",
+      name: "Hackathon Grants",
       icon: <Icon glyph="sam" size={16} />,
       perform: () =>
         (window.location.href = adminUrls["FIRST Grants"]),
```
Comment (sampoder) [inline app/javascript/components/command_bar/CommandBar.js]: Let's also update the admin URLs hash to use Hackathon Grants

**Review by sampoder — CHANGES_REQUESTED** [review]
Awesome! Could we also update the card on the homepage?

Screenshot 2023-12-22 at 10 59 42 AM

File: app/javascript/components/command_bar/CommandBar.js (hunk 1 above)
Comment (aramshiva) [reply app/javascript/components/command_bar/CommandBar.js]: oops my bad!

**Review by sampoder — APPROVED** [review]
LGTM!

**Comment by aramshiva** [comment]
just fixed conflicts!

**Review by garyhtou — APPROVED** [review]
coolio! 1️⃣🤖

**Pull request merged**